docker/dockerfile-joex

```diff
@@ -0,0 +1,45 @@
+FROM docspell-dev:base
+
+ENV UNO_URL https://raw.githubusercontent.com/unoconv/unoconv/0.9.0/unoconv
+
+RUN apk add --no-cache openjdk11-jre \
+    bash \
+    curl \
+    ghostscript \
+    tesseract-ocr \
+    tesseract-ocr-data-deu \
+    unpaper \
+    wkhtmltopdf \
+    libreoffice \
+    ttf-droid-nonlatin \
+    ttf-droid \
+    ttf-dejavu \
+    ttf-freefont \
+    ttf-liberation \
+    libxml2-dev \
+    libxslt-dev \
+    pngquant \
+    zlib-dev \
+    g++ \
+    qpdf \
+    py3-pip \
+    python3-dev \
+    libffi-dev\
+    qpdf-dev \
+    openssl-dev \
+    ocrmypdf \
+  && pip3 install --upgrade pip \
+  && pip3 install ocrmypdf \
+  && curl -Ls $UNO_URL -o /usr/local/bin/unoconv \
+  && chmod +x /usr/local/bin/unoconv \
+  && apk del curl libxml2-dev libxslt-dev zlib-dev g++ python3-dev py3-pip libffi-dev qpdf-dev openssl-dev
+
+RUN ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
I just saw, that I really removed this line from the joex-base dockerfile… This makes me wonder if the current version can process docx/odt files at all…